specification/securityinsights/resource-manager/readme.md

```diff
@@ -45,6 +45,18 @@ directive:
 
 ---
 
+### Tag: package-2021-03-preview-only
+
+These settings apply only when `--tag=package-2021-03-preview-only` is specified on the command line.
+
+```yaml $(tag) == 'package-2021-03-preview-only'
+input-file:
+- Microsoft.SecurityInsights/preview/2021-03-01-preview/Settings.json
```

Comment:
Is SecurityInsights.json in stable still valid? If so, please include in this as well.  Same as above.

Reply:
for 2021-03, we only have settings

Reply:
Hi @yuvalshilo I have similar question: could we introduce a tag that could consist of all functionality in this RP?
The `package-composite-v1` has the name of `composite`, you could add new swaggers into it to fulfill the functionality so that the tag `package-composite-v1` could be a full package of all the functionalities in this RP.

Reply:
It's not possible because the previous version contains endpoints that exist in the new version.
But - we are currently working on moving all of our endpoints to the new version, and it will be done in the next few months.